content/sensu-go/6.8/release-notes.md

```diff
@@ -110,6 +111,32 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.8.2 release notes
+
+**October 6, 2022** &mdash; The latest release of Sensu Go, version 6.8.2, is now available for download.
+
+Sensu Go 6.8.2
+
+Read the [upgrade guide][1] to upgrade Sensu to version 6.8.2.
+
+**IMPROVEMENTS:**
+
+- The authentication module now logs successful (INFO) and unsuccessful (ERROR) login attempts.
+- Agent websocket connection logging includes backend entity name.
+- A sensu.io/output_truncated_bytes label is now applied to events when the check output has been truncated due to a check's max_output_size configuration.
```

Comment:
```suggestion
- When check output is truncated due to the [max_output_size][303] configuration, the events the check produces will include a `sensu.io/output_truncated_bytes` label.
```

Reply:
If this change is accepted, you'll also need to add this link reference:

[302]: /sensu-go/6.8/observability-pipeline/observe-schedule/checks/#check-output-truncation-attributes